### PR TITLE
chore(curriculum): validate ferris wheel project 

### PR DESCRIPTION
**Pull request by Liam-boston**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Contributes to #46540

### CODE REVIEW

**Review by Sboonny — DISMISSED** [review]
#### :sparkles: :tada: **AWESOME!** :tada: :sparkles:

 Hi @Liam-boston, thanks for this pull request and for contributing to the code-base for the first time. We are looking forward to more contributions from you in the future.

 Cheers & happy contributing!

File: curriculum/challenges/english/14-responsive-web-design-22/learn-css-animation-by-building-a-ferris-wheel/6169b284950e171d8d0bb16a.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ assert(new __helpers.CSSHelp(document).getCSSRules('keyframes')?.[1]?.cssRules?.
 
 ```html
 <!DOCTYPE html>
-<html>
+<html lang="en">
```
Comment (moT01) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-css-animation-by-building-a-ferris-wheel/6169b284950e171d8d0bb16a.md:41]: This is the last step, it includes a solution at the bottom of the file. Can you add this `lang="en"` to the solution @Liam-boston

Comment (Liam-boston) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-css-animation-by-building-a-ferris-wheel/6169b284950e171d8d0bb16a.md:41]: Yup np

**Review by Sboonny — APPROVED** [review]
LGTM✅

**Review by Sembauke — APPROVED** [review]

**Pull request merged**